### DISCUSSION TITLE
A place to exchange ideas and keep in touch

### DISCUSSION THREAD

**Discussion by rcatlord** [tech-acs/crvsreportpackage#20] [Announcements]
Hi everyone,

[GitHub Discussions](https://resources.github.com/devops/process/planning/discussions) seems to operate much like Slack or Microsoft Teams as a place to share ideas, present feedback and make announcements. Unlike GitHub Issues which generally concern changes to the codebase, GitHub Discussions is more open-ended. I invite everyone who is contributing to the CRVS report project to ask questions, post updates, and share best practice.

Henry

**Comment by diego-ons** [comment]
Can you see this: @tbelay , @gulilattesfaye , @jeroenminderman , @beardl-ons ?
